Makefile.toml

```diff
@@ -5,7 +5,7 @@ additional_profiles = [ "watch" ]
 
 # component builds
 [tasks.all]
-dependencies = [ "tools", "backend", "frontend" ]
+dependencies = [ "tools", "backend", "frontend", "lobby" ]
```

Comment:
I suppose we keep the crates separate then? Just FMI

Reply:
Haven't taken a final decision yet: this change was in my working tree for a while and I decided to push it for now. However I got some doubts whether we *actually* want to do this, but I'll think tomorrow about this and ping you after that, ok? :)

Reply:
Alright, thx

Reply:
So, after reconsidering this, I came to the conclusion that we preferably want to use two crates for the following reasons:

(1) both servers would log to `stderr` and having outputs from both in `docker-compose logs` is IMHO harder to parse mentally.
(2) we could simply share the sqlite-db with both containers and use the game-lobby's Rust-API in rask-server to validate a token.

Right now I'm wondering though if we actually want to move this to `squirrel-labs/GameLobby` and use e.g. a docker-container and include the Git repo in rask-server's `Cargo.toml`.

In that case I'd still leave this PR open, first of all to have one place where we can review the entire Rust code and in the end we'd still have to modify the rask-server. As soon as the changes are usable, I'd move the `lobby/` subtree to `GameLobby`.

Does this sound reasonable to you?

Reply:
Sounds good, I'm fine with keeping them seperate, I think that might even be cleaner.
And the development is a little easier if both projects are in the same repo
Shouldn't we still rename the rask engine to game-engine though?

Reply:
I don't think so: while those crates partly implement stuff for the game-lobby, those are primarily designed for Ratatosk and should be named accordingly :)